.travis.yml

```diff
@@ -10,18 +10,34 @@ addons:
     update: true
 
 rust:
-  - nightly
-  - beta
   - stable
   - 1.46.0
 
-jobs:
-  allow_failures:
-    - rust: nightly
-
 before_install:
-  - sudo apt-get -y install libfontconfig1-dev
+  - sudo apt-get -y install libfontconfig1-dev jq
 
 script:
   - cargo test
+  - |
+    if [ "$TRAVIS_RUST_VERSION" = "stable" ]; then
+      ./ci/ldd-grep
+    else
+      true
+    fi
+
+  # Rust 1.46.0 cargo doesn't accept carget test --features
+  - cargo test --features dlopen --manifest-path fontconfig/Cargo.toml
+  - |
+    if [ "$TRAVIS_RUST_VERSION" = "stable" ]; then
+      ./ci/ldd-grep -v -- --features dlopen
+    else
+      true
+    fi
+
   - RUST_FONTCONFIG_DLOPEN=on cargo test
```

Comment:
Sorry, I've never seen this before. What is `on`?

Also, you set `RUST_FONTCONFIG_DLOPEN` but when is it used? Google seems to not find anything for it, even in doublequotes.

Also, TIL you can have newlines and pipes in YAML.

Reply:
Ignore me... my mouse scroll wheel wasn't working and so didn't see the rest of the PR :headslap:

Reply:
> Also, TIL you can have newlines and pipes in YAML.

https://yaml-multiline.info/ is very handy